src/Arcus.Template/XunitTestLogger.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using GuardNet;
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace Arcus.Testing.Logging
```

Comment:
Shouldn't this file be in `src/Arcus.Template.Logging/XunitTestLogger.cs`?

Reply:
Ah, yes, have forgotton to change to folder structure.